src/openforms/submissions/logic/log_utils.py

```diff
@@ -22,3 +29,13 @@ def get_targeted_components(
         targeted_components.append(action_log_data)
 
     return targeted_components
+
+
+def evaluate_json_logic(logic: JSONObject, data: dict, rule: FormLogic) -> Any:
+    try:
+        return jsonLogic(logic, data)
+    except Exception as error:
+        logger.error(
+            "Error in rule %s: evaluation of %s raised error: %s", rule.pk, logic, error
+        )
```

Comment:
```suggestion
        logger.error(
            "Error in rule %s: evaluation of %s failed", 
            rule.pk, 
            logic, 
            exc_info=error
        )
```
this will attach the traceback and include the info for sentry.